usr.bin/man/man.sh

```diff
@@ -1,30 +1,30 @@
-#! /bin/sh
-#
+#!/bin/sh
+#-
```

Comment:
is this in a style-guide anywhere? or used elsewhere?

Reply:
Yes, https://docs.freebsd.org/en/articles/committers-guide/#pref-license